mustache-templates/upgrade-docker.yml.mustache

```diff
@@ -8,10 +8,8 @@ services:
       "--env-type", "{{envType}}{{^envType}}standalone{{/envType}}",
       "--license", "{{license}}"
     ]
-    {{#skip_pull_image}}
     environment:
-      - SKIP_PULL=1
-    {{/skip_pull_image}}
+      - SKIP_PULL={{skip_pull_image}}{{^skip_pull_image}}0{{/skip_pull_image}}
```

Comment:
is this change needed? in portainer-updater we check if SKIP_PULL is 1. if it's missing, it should be ok, no? no need for a default